contracts/modules/STO/ISTO.sol

```diff
@@ -3,30 +3,16 @@ pragma solidity ^0.4.24;
 import "../../Pausable.sol";
 import "../Module.sol";
 import "../../interfaces/IERC20.sol";
+import "./ISTOStorage.sol";
 import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 
 /**
  * @title Interface to be implemented by all STO modules
  */
-contract ISTO is Module, Pausable  {
+contract ISTO is ISTOStorage, Module, Pausable  {
     using SafeMath for uint256;
 
     enum FundRaiseType { ETH, POLY, DAI }
```

Comment:
Do enums affect storage?

Reply:
As per my knowledge nope, they are only the declaration

Reply:
Just tested, enums and public constants don't affect storage layout.

Reply:
Same for Enums 
Test gist - 
https://gist.github.com/satyamakgec/ca55f663480828e5eff7c6b9d1ae83b2